api/advanced-api.yaml

```diff
@@ -0,0 +1,114 @@
+paths:
+  /advanced/config:
+    post:
+      tags: ['Advanced']
+      operationId: set_advanced_config
+      description: |-
+        An API that extends the base Traffic Generator configuration with 
+        advanced feature configuration.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '../advanced/config.yaml#/components/schemas/Advanced.Config'
+      responses:
+        '200':
+          $ref: '../result/request.yaml#/components/responses/Success'
+        '409':
+          $ref: '../result/request.yaml#/components/responses/Conflict'
+        '590':
+          $ref: '../result/request.yaml#/components/responses/Fail'
+        '591':
+          $ref: '../result/request.yaml#/components/responses/NotSupported'
+
+  /advanced/metrics:
+    post:
+      tags: ['Advanced']
```

Comment:
Should the tag be 'Advanced Metrics'?

Reply:
Keeping all advanced APIs under the tag Advanced rather than two separate tags for config and metrics. This can easily be changed in the future without incurring any compatibility issues.